### PR TITLE
update google issue query to handle field searches

### PR DESCRIPTION
**Pull request by pgrace-google**

### CODE REVIEW

File: src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py [hunk 1]
```diff
@@ -1122,7 +1122,11 @@ def _parse_datetime(date_string):
 
 def _get_query(keywords, only_open):
   """Gets a search query."""
-  query = ' '.join('"{}"'.format(keyword) for keyword in keywords)
+  string_keywords = [word for word in keywords if ':' not in word]
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py:1125]: Can we make this function accept two kinds of keywords so we don't need this logic in this function?

Comment (pgrace-google) [reply src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py:1125]: you know what i love that

Comment (pgrace-google) [reply src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py:1125]: nvm i take it back - that means we would have to adjust the interface function for find_issues and find_issues_url

but we also would only really have to update the google_internal_tracker function implementations so i guess its not that bad

Comment (pgrace-google) [reply src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py]: ok nvm lint is failing i gotta update monorail and jira too but now im feeling the sunk cost fallacy so im gonna update it anyway

Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py]: Feel free to disable (using pylint) or delete anything in monorail that's failing. It's totally dead code.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**